### DISCUSSION TITLE
Consider the Eclipse 1.0 license for Lingy

### DISCUSSION THREAD

**Discussion by ingydotnet** [ingydotnet/lingy#27] [General]
We currently use the MIT license which I think is compatible.

This issue is primarily for public discussion on the matter.

**Comment by PEZ** [comment]
I was recommended to watch this presentation about Open Source Licenses for Developers, from London Clojurians earlier this year: https://www.youtube.com/watch?v=m478BHGR3XU

**Comment by PEZ** [comment]
And here as well: https://www.juxt.pro/blog/prefer-mit/

**Comment by ingydotnet** [comment]
Thanks. Watched and read both.
Lingy will at some point need to use all the Clojure .clj source files.
That might force the issue.
